src/Exception/NoContentResponseException.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Exception to send an empty response and exit the execution in the Contao workflow.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class NoContentResponseException extends AbstractResponseException
```

Comment:
The naming here reflects the HTTP status code which is `204 No Content`.
I tried to stay as close as possible to the HTTP status codes instead of making up arbitrary names from which you can not guess the status code.
Regarding the comment, I think "Exception to send an empty response" is easier to read than "Exception to send a response without content" or "Exception to send a response with no content".

@Toflar and @leofeyer what do you think?